Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/satchel.yml

```diff
@@ -244,4 +244,10 @@
     contents:
     - id: GasAnalyzer
     - id: trayScanner
+    - id: Multitool
+    - id: AccessConfiguratorUniversal
+    - id: HandLabeler
+    - id: PowerDrill
+    - id: JawsOfLife
+    - id: WelderExperimental
```

Comment:
Please add comments indicating that those changes belong to floofstation. Simplest way it to suffix each line with `# Floofstation`, a more complex way could involve sectioning this part of the file off and indicating that the whole section belongs to floof (e.g. `# Floofstation section - expanded AGhost inventory` followed by `# Floofstation section end`).

This is necessary to make future editing, debugging, and upstream merging easier.